resources/standards-data/ldml-keyboards/techpreview/fetch-latest-cldr-techpreview.sh

```diff
@@ -59,6 +59,10 @@ rm -rf ./import ./3.0 ./dtd ./test
 # copy over everything
 cp -Rv "${IMPORT_DIR}" "${DATA_DIR}" "${DTD_DIR}" "${TEST_DIR}" .
 
+# delete old files, no reason to keep them
+rm -vf dtd/{ldmlKeyboard,ldmlPlatform}.{xsd,dtd}
```

Comment:
This won't be needed for long, of course

Reply:
```suggestion
# delete old draft spec files, no reason to keep them
rm -vf dtd/{ldmlKeyboard,ldmlPlatform}.{xsd,dtd}
```

Reply:
It will because these are the files requested to be re added to CLDR. The pre 3.0.